fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/ReindexOperationPhase2Test.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.server.test.operation;
+
+import static com.ibm.fhir.model.type.String.string;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.UUID;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.core.FHIRMediaType;
+import com.ibm.fhir.model.parser.exception.FHIRParserException;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.OperationOutcome;
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.test.TestUtil;
+import com.ibm.fhir.path.exception.FHIRPathException;
+import com.ibm.fhir.server.test.FHIRServerTestBase;
+
+/**
+ * This integration tests the <b>fhir-operation-reindex</b> $reindex operation.
+ * Phase 2: Completes the reindex changes
```

Comment:
Not quite sure what this means.  It looks like it performs a number of searches and confirms we get results.
What doesn't it do?

Reply:
Updated to  * <b>Phase 2</b>:
 * Pre-condition: The Phase 1 loaded Resources which were not previously found.
 * Completes the reindexes with the newly found SearchParameters.
 * These SearchParameters include the original FHIR Path Expressions important for the `ChangedExpression`.
 *
 * This test is followed by a Server Side Restart and then Phase3.